community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+<a name="civilized"></a>
+
+## [This is a Civilized Place for Public Discussion](#civilized)
+
+Please treat this discussion forum with the same respect you would a public park. We, too, are a shared community resource &mdash; a place to share skills, knowledge and interests through ongoing conversation.
+
+These are not hard and fast rules. They are guidelines to aid the human judgment of our community and keep this a kind, friendly place for civilized public discourse. Please read this document carefully to avoid accidentally violating these guidelines.
+
+If we see posts that we judge to not meet these guidelines, we will actively remove them, but will also let you know privately why we're doing so. 
+
+Everything in this forum is also covered by our [Code of Conduct](https://exercism.org).
+
+<a name="encourage"></a>
+
+## [Encourage each other](#improve)
+
+This is first and foremost an educational space - a space to learn and share ideas, to uplift each other. It is **not** a space to prove yourself, show off your knowledge, compete with others, create a sense of “us vs them”, or put others down. We take a strict approach to this, and so topics that would be considered ok in other forums are not considered ok here. 
+
+Some examples:
+- A discussion exploring what the tradeoffs would be between using C# vs F# could be a positive and educational experience. However, a comment saying “C# is rubbish because …” is not helpful, does not add to the educational experience and would be deleted.
+- Any conversations around topics such as politics or religion are not appropriate in this space. Neither are topics that are fought with a religious fervor (e.g. whether VIM or Emacs is better).
+- Any sort of comment that implies “noobs do …” or “real developers use …” will be deleted.
+
+<a name="improve"></a>
+
+## [Improve the Discussion](#improve)
+
+Help us make this a great place for discussion by **always adding something positive to the discussion**. If you are not sure your post adds to the conversation, think over what you want to say and try again later.
+
+One way to improve the discussion is by discovering ones that are already happening. Spend time browsing the topics here before replying or starting your own, and you’ll have a better chance of meeting others who share your interests.
+
+The topics discussed here matter to us, and we want you to act as if they matter to you, too. Be respectful of the topics and the people discussing them, even if you disagree with some of what is being said.
```

Comment:
"adding something **positive** to the discussion" doesn't match the rest of the paragraph imo. A post can be negative, e.g. because it points out an issue with an approach or whatever, but still be valuable and useful for the discussion.

Reply:
What about `always adding something valuable to the discussion`?

Reply:
I think that's better